SWLOR.Game.Server/Core/NWScript/Enum/FeatType.cs

```diff
@@ -1850,6 +1850,7 @@ public enum FeatType
         GreaterEarthquake = 1883,
         FlameBlast = 1884,
         Infusion1 = 1885,
-        Infusion2 = 1886
+        Infusion2 = 1886,
+        ZenMarksmanship = 1887
```

Comment:
As mentioned in the hak PR, please adjust this to 1995 to account for incoming Beast Mastery changes.

Reply:
Amended.